cli/create/templates/app_files.go

```diff
@@ -1,6 +1,8 @@
 package templates
 
-import "github.com/tarantool/cartridge-cli/cli/templates"
+import (
+	"github.com/tarantool/cartridge-cli/cli/templates"
+)
 
 var appFilesTemplate = templates.FileTreeTemplate{
```

Comment:
I think that now we don't need to declare `templates.FileTreeTemplate` manually. 
You can scan `content` directory tree and collect all files content, mode and path.

Imagine, that you've added a new file to application template. I think it would be great if adding a new file to templates directory is enough.

So, I suggest creating some directory (e.g.`./cli/create/templates/cartridge`, `cartridge` is a template name, it mean Cartridge application) that is exactly the same as a result application directory (except some templates like`{{ .Name }}`).